sequoia-openpgp doesn't compile without any features being enabled, rely on the
default feature being sane.

Index: sequoia-autocrypt/Cargo.toml
===================================================================
--- sequoia-autocrypt.orig/Cargo.toml
+++ sequoia-autocrypt/Cargo.toml
@@ -63,7 +63,6 @@ version = ">= 0.21, < 0.23"
 
 [dependencies.sequoia-openpgp]
 version = "2"
-default-features = false
 
 [target."cfg(not(windows))".dev-dependencies.sequoia-openpgp]
 version = "2"
